actnow/petitions/models.py

```diff
@@ -70,3 +76,15 @@ def __str__(self):
 
     class Meta:
         unique_together = ("petition", "signatory")
+
+
+class Source(TimestampedModelMixin):
+    link = models.URLField(_("link"), unique=True)
+
+    def __str__(self):
+        return self.link
+
+    def clean(self):
+        while self.link.endswith("/"):
+            self.link = self.link[:-1]
+        return super().clean()
```

Comment:
I think what we were talking about is called URL normalisation. We could use something like in in this [gist](https://gist.github.com/maggyero/9bc1382b74b0eaf67bb020669c01b234) or the [hiperlink](https://hyperlink.readthedocs.io/en/latest/api.html#hyperlink.URL.normalize) library